db/schema.rb

```diff
@@ -10,9 +10,10 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_09_29_190236) do
+ActiveRecord::Schema.define(version: 2020_10_06_234242) do
 
   # These are extensions that must be enabled in order to support this database
+  enable_extension "hstore"
```

Comment:
how does this keep ending up here?

Reply:
I dunno, I haven't yet found documentation on why it's auto-added.  My random guess is that storing hashes into the DB as a JSON/YML string enables `hstore`?

If you think it's a problem, I'm happy to remove it.